### ISSUE TITLE
Last vowel is not deleted in кала

### ISSUE THREAD

**Issue by hectoralos** [apertium/apertium-chv#15]
In some verbal forms the last а of verb кала is not deleted:

```
$ echo "кала<v><iv><fut><p1><sg>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><fut><p1><sg>	кала>{Ă}п	0,000000
```

_калаӑп_ is generated instead of _калӑп_

```
$ echo "кала<v><iv><fut><p2><sg>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><fut><p2><sg>	кала>{Ă}н	0,000000
```

_калан_ is generated  instead of _калӑн_

> $ echo "кала<v><iv><fut><p1><pl>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
> кала<v><iv><fut><p1><pl>	кала>{Ă}п{Ă}р	0,000000

_калаӑпӑр_ is generated instead of _калӑпӑр_

```
$ echo "кала<v><iv><fut><p2><pl>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><fut><p2><pl>	кала>{Ă}р	0,000000
```
_калаӑр_ is generated instead of _калӑр_

```
$ echo "кала<v><iv><cond><p1><sg>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><cond><p1><sg>	кала>{Ă}тт{Ă}м	0,000000
```
_калаӑттӑм_ is generated instead of _калӑттӑм_

```
$ echo "кала<v><iv><cond><p2><sg>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><cond><p2><sg>	кала>{Ă}тт{Ă}н	0,000000
```
_калаӑттӑн_ is generated instead of _калӑттӑн_

```
$ echo "кала<v><iv><cond><p1><pl>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><cond><p1><pl>	кала>{Ă}тт{Ă}м{Ă}р	0,000000
```
_калаӑттӑмӑр_ is generated instead of _калӑттӑмӑр_

```
$ echo "кала<v><iv><cond><p2><pl>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><cond><p2><pl>	кала>{Ă}тт{Ă}р	0,000000
```
_калаӑттӑр_ is generated instead of _калӑттӑр_

```
$ echo "кала<v><iv><imp><p1><sg>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><imp><p1><sg>	кала>{A}м	0,000000
```

_калаам_ is generated instead of _калам_

```
$ echo "кала<v><iv><imp><p2><pl>" | hfst-lookup .deps/chv.LR.lexc.hfst 2> /dev/null
кала<v><iv><imp><p2><pl>	кала>{Ă}р	0,000000
```

_калаӑр_ is generated instead of _калӑр_

**Comment by ftyers** [comment]
Hmm, before I start with this, I want to check, is the `<fut><p2><sg>` the same form as the `<gen>` ?

```
%<p2%>%<sg%>:%>%{Ă%}н
%<gen%>:%>%{Ă%}н
```

If so, do different things happen with these to vowel-ending stems? e.g.
```
$ echo "хала<n><gen>" | hfst-lookup chv.autogen.hfst 
хала<n><gen>	халан	0,000000
```
Is this correct? If so, our morphotactic form will need to be different.

**Comment by jonorthwash** [comment]
Shouldn't `{Ӑ}` just always get deleted after a non-"short" vowel?  Are there any examples where this isn't the case?

**Comment by jonorthwash** [comment]
Oh, I see, there's `{Ӑ}` that should take over the previous vowel and I guess there should be `{ӑ}` that gets deleted.  And I guess `{А}` should be `{а}` instead, too.

**Comment by jonorthwash** [comment]
Btw, there's a pretty old yaml file with these tests already in it: https://github.com/apertium/apertium-chv/blob/master/tests/v_%D0%BA%D0%B0%D0%BB%D0%B0_del.yaml

Total passes: 12, Total fails: 35, Total: 47

**Comment by hectoralos** [comment]
@ftyers, you are absolutely right. The rules are different. In <fut>, <cond> and <ccond> we should not use {Ă} but something else. In these tenses the Ă from the tense suffix takes precedence over a final А in the stem. (I'll soon create a new issue for <ccond>, because there is a similar problem for p3.sg. I have to do a few last checks)

**Comment by hectoralos** [comment]
I now see that the <ccond> tense I created is the existing <opt> tense, which has quite a lot of problems. I'll delete the <ccond> tense, maybe modify a bit <opt> and open a specific issue for it.

**Comment by hectoralos** [comment]
@ftyers , wait! I see something odd in conditional. I want to confirm the endings.

**Comment by hectoralos** [comment]
@ftyers , everything in the description of the issue seems to be right. The problems I saw seem to be all related to <abil>. I am creating new yaml-files and adding more forms to the existing ones.
